news/2025/2025-05-11-resurrection-cup-2025-registrations-now-open.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Resurrection Cup 2025: Registrations Now Open!"
+date: 2025-05-11 12:00:00 +0000
+---
+
+The renaissance is upon us with Resurrection Cup returning for a 2025 iteration — with more bangers, more customs and a large focus on competitivity!
+
+![](/wiki/shared/news/2025-05-11-resurrection-cup-2025-registrations-now-open/banner.jpg)
+
+[Resurrection Cup](https://osu.ppy.sh/community/forums/topics/20767782), or ResCup for short, is an annual open-rank Team Vs tournament that attempts to bring a breath of fresh air to the top-tier competitive tournament scene.
+
+## Original music and original maps
+
+Over the past couple years, we have had the joy of sharing 40 originals with you. You can expect that trend to continue this year as well since, after the Qualifiers have finished, we will be releasing multiple original songs in every single mappool. If you thought last year was good, you bet we are going to surpass expectations this year as well, so be sure to tune in.
+
+If you haven't seen it yet, here is a crossfade of the upcoming *Wishes for Tomorrow* album which will include 22 of them!
```

Comment:
this was announced in a news post last year, and to copy-paste the wording devoid of that context and without any news sounds off

```suggestion
22 of last year's songs will also later be released as part of an album. To get a feeling of what this tournament has to offer sound-wise, here's a sample:
```